tests/ccl_sample_pkemu.c

```diff
@@ -0,0 +1,44 @@
+#include "ccl.h"
+#include <math.h>
+#include <stdio.h>
+#include "ccl_params.h"
+
+int main(int argc, char * argv[])
+{
+  int status=0;
+  double Omega_c = 0.25;
+  double h = 0.7;
+  double Omega_b = 0.022/h/h;
+  double normp = 0.8; //2.1e-9
+  double n_s = 0.96;
+  double Omega_k = 0.;
+  double Neff = 3.04;
+  double Nmass = 1;
```

Comment:
I think that in speaking to Katrin we established that the emulator requires the total number of neutrinos to be fixed to 3.04. This sets the total number to 4.04 so I think it will give incorrect results.